docs/sinks/redis.md

```diff
@@ -0,0 +1,16 @@
+# Redis Sink
+
+### Input Schema
+Redis  sink in Depot only supports Protobuf schema as of now. Support for JSON and Avro format is planned for a future release of Depot.
+
+### Data Types
+Redis sink can be created in 2 different modes based on the value of [`SINK_REDIS_DATA_TYPE`](../sinks/redis-sink.md#sink_redis_data_type): HashSet or List
+  - `Hashset`: For each message, an entry of the format `key : field : value` is generated and pushed to Redis. field and value are generated on the basis of the config [`INPUT_SCHEMA_PROTO_TO_COLUMN_MAPPING`](../sinks/redis-sink.md#-input_schema_proto_to_column_mapping-2)
```

Comment:
`INPUT_SCHEMA_PROTO_TO_COLUMN_MAPPING` this config is not present. 
can you rephrase it?

Reply:
done